src/System/Requirement/InstallationNotOverriden.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2022 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\System\Requirement;
+
+use FilesystemIterator;
+
+final class InstallationNotOverriden extends AbstractRequirement
+{
+    public function __construct()
+    {
+        $this->title = __('Installation overrides detection');
+    }
+
+    protected function check()
+    {
+        $version_folder = GLPI_ROOT . '/.version/';
+        try {
+            $file_iterator = new FilesystemIterator($version_folder);
+        } catch (\UnexpectedValueException $e) {
+            $this->validated = false;
+            $this->validation_messages[] = __("`.version` folder doesn't exist.");
+            $this->optional = true;
```

Comment:
Removal of `.version` directory is really an edge case. It would probably happen only if someone distribute an altered archive of GLPI or if an administrator manually deleted this file before running the install/update process.
I think we should just consider that this requirement have to be skipped in such case.

```suggestion
            $this->out_of_context = true;
```